### DISCUSSION TITLE
Problem when integrating an SVG image

### DISCUSSION THREAD

**Discussion by Leopold194** [py-pdf/fpdf2#715] [Q&A] [answered]
I am trying to embed a .svg image in my pdf using the line: 
`pdf.image(f"{pathlib.Path(__file__).parent}/../../assets/pictures/Ban_Historique_Maladie_Aveclogo.svg")`

But I get the following error that I can't find on the Internet. 
`raise ValueError(f"unsupported color specification {colorstr}")
ValueError: unsupported color specification url(#paint0_linear_137_38)`

I put the image in attachment

![Ban_Historique_Maladie_Aveclogo](https://user-images.githubusercontent.com/46968525/223703937-34da1b73-17c2-4d3d-84f8-b77a024758c6.svg)

**Comment by Lucas-C** [comment, marked as answer]
Hi @Leopold194!

If you open this file with a text editor, you will see thi one the line 14:
```svg
fill="url(#paint0_linear_137_38)
```
This is a reference to the `def` on line 3:
```svg
<linearGradient id="paint0_linear_137_38" x1="0" y1="46.63" x2="595.28" y2="46.63" gradientUnits="userSpaceOnUse">
```
`url()` as `fill` values are part of the SVG feature that `fpdf2` currently does not support:
* https://pyfpdf.github.io/fpdf2/SVG.html#currently-unsupported-notable-svg-features
* issue https://github.com/PyFPDF/fpdf2/issues/537

Currently, the best I can suggest is either:
* replace `url(#paint0_linear_137_38)` by `#56B35D` in your SVG source image, and `fpdf2` will be able to insert it, but you will loose the gradient effect.
* contribute to `fpdf2` and add support for this feature! 😊

**Reply by Leopold194** [reply]
Thanks for the help !
